### ISSUE TITLE
Add valid example data package in repository

### ISSUE THREAD

**Issue by peterdesmet** [tdwg/camtrap-dp#118]
Add small valid example data package in repository

For a small example (not all columns, see https://github.com/tdwg/dwc-for-biologging/tree/403f57db105982dc05b70f3cf66fd2b5591798db/derived/camtrap-dp/data/raw

**Comment by peterdesmet** [comment]
I think it would be good to have a real world example, based on https://doi.org/10.5281/zenodo.4893244

**Comment by peterdesmet** [comment]
Here's a reproducible way to select representative data from the published dataset:

```R
library(tidyverse)
library(datapackage)

# Source data + selection
descriptor <- "https://zenodo.org/record/4893244/files/datapackage.json"
deployments <- c(
  "577b543a-2cf1-4b23-b6d2-cda7e2eac372",
  "7ca633fa-64f8-4cfc-a628-6b0c419056d7",
  "62c200a9-0e03-4495-bcd8-032944f6f5a1",
  "29b7d356-4bb4-4ec4-b792-2af5cc32efa8"
)

# Download selection
dir.create("example")
download.file(descriptor, "example/datapackage.json")
pkg <- read_package(descriptor)
for (name in pkg$resource_names) {
  data <- read_resource(pkg, name)
  data <- data %>% filter(deployment_id %in% deployments)
  write_csv(data, paste0("example/", name, ".csv"), na = "")
}

# Note that the datapackage.json selector needs to be manually trimmed
# to the scope of the selection
```

**Comment by peterdesmet** [comment]
Solved with #175.
